Utils/proxy.py

```diff
@@ -146,18 +150,26 @@ def restore_default():
     shutil.copy(apt_backup, apt_)
     shutil.copy(env_backup, env_)
     shutil.copy(bash_backup, bash_)
+    shutil.copy(docker_backup, docker_)
 
 
 
 if __name__ == "__main__":
 
-    # create backup	if not present
+    # create backup     if not present
+    if not os.path.isfile(apt_):
+        open(apt_, 'a').close()
+    if not os.path.exists(docker_path):
+        os.makedirs(docker_path)
+    if not os.path.isfile(docker_):
+        open(docker_, 'a').close()
+
     if not os.path.isdir("./.backup_proxy"):
         os.makedirs("./.backup_proxy")
-        if os.path.isfile(apt_):
-            shutil.copyfile(apt_, apt_backup)
-        shutil.copyfile(env_, env_backup)
-        shutil.copyfile(bash_, bash_backup)
+    shutil.copyfile(apt_, apt_backup)
+    shutil.copyfile(env_, env_backup)
+    shutil.copyfile(bash_, bash_backup)
+    shutil.copyfile(docker_, docker_backup)
```

Comment:
There is a problem with indentation.

Reply:
I've reformat code by PyCharm linter. Please check.